### PR TITLE
add: new icons and table no data component

### PR DESCRIPTION
**Pull request by Jimmy-Joseph19**
Updates for ui-components

1. New icons and no data component for table

### CODE REVIEW

File: packages/ui-components/src/components/UIDialog/UIDialog.tsx [hunk 1]
```diff
@@ -206,6 +206,14 @@ export class UIDialog extends React.Component<DialogProps, DialogState> {
      */
     render(): JSX.Element {
         const {
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            acceptButtonText,
```
Comment (devinea) [inline packages/ui-components/src/components/UIDialog/UIDialog.tsx:210]: Why is unused variable needed here?

**Review by 815are — APPROVED** [review]

**Review by devinea — APPROVED** [review]
Changes look good, tests and story updated. 
Thanks @Jimmy-Joseph19

File: packages/ui-components/src/components/UIDialog/UIDialog.tsx (hunk 1 above)
Comment (815are) [reply packages/ui-components/src/components/UIDialog/UIDialog.tsx:210]: @devinea 

Full code is:
```
const {
            // eslint-disable-next-line @typescript-eslint/no-unused-vars
            acceptButtonText,
            cancelButtonText,
            onAccept,
            onCancel,
            acceptButtonId,
            cancelButtonId,
            footer,
            closeButtonVisible,
            multiLineTitle,
            scrollArea = UIDialogScrollArea.Content,
            children,
            ...rest
        } = this.props;
```

then `rest` is used to deepMerge properties. In summary by adding unused variable we exclude it from `rest`

Comment (Jimmy-Joseph19) [reply packages/ui-components/src/components/UIDialog/UIDialog.tsx:209]: I extended the [eslintrule](https://github.com/SAP/open-ux-tools/pull/692/files#diff-c42f01ebf3490ee1be748e94d235263e6c5c0d7dd09ba3429788f5e214c519a8R23) to `ignoreRestSiblings` so i removed the inline comment to ignore `unusedvars`.

Comment (devinea) [reply packages/ui-components/src/components/UIDialog/UIDialog.tsx:209]: Thanks for info and update.

**Review by devinea — APPROVED** [review]

**Review by 815are — APPROVED** [review]
looks ok.
Did not test locally - code was already tested with internal version of ts

**Pull request merged**